2837c92 : Catch the fs::exists() exception when attempting to write the patch (#7514) - EvilDragon